mitodl/hugo-course-publisher
# Course Info Collapsibles

#### Pre-Flight checklist


#### What are the relevant tickets?
https://trello.com/c/kVtunwvL/33-course-info-multiple-departments-multiple-course-numbers

#### What's this PR do?
Implements a new partial for partially collapsible lists and uses it on the relevant items in Course Info.

#### How should this be manually tested?
Run the site, check out Course Info on various courses and make sure it looks / functions right.

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/12089658/75587643-e9805880-5a44-11ea-95ae-051e300a518a.png)

![image](https://user-images.githubusercontent.com/12089658/75587682-074dbd80-5a45-11ea-9cdf-68b2e0847bba.png)
